tests/gpt2/test_modeling_tf_gpt2.py

```diff
@@ -294,6 +294,19 @@ def create_and_check_gpt2_lm_head(self, config, input_ids, input_mask, head_mask
         result = model(inputs)
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.seq_length, self.vocab_size))
 
+    def create_and_check_gpt2_xla_generate(self, config, *args):
```

Comment:
check that xla and non-xla is always the same. Using the same input_ids here since it's a bit flaky (1/20 runs there seem to be some numerical small differences) and because XLA doesn't cut output ids in case EOS token is generated.